### PR TITLE
PICARD-384: Allow wildcards for unsetting certain tags

### PR DESCRIPTION
**Pull request by samj1912**

### CODE REVIEW

File: picard/script.py [hunk 1]
```diff
@@ -413,6 +413,13 @@ def func_unset(parser, name):
     """Unsets the variable ``name``."""
     if name.startswith("_"):
         name = "~" + name[1:]
+    # Allow wild-card unset for certain keys
+    if name in ('performer:*', 'comment:*', 'lyrics:*'):
```
Comment (zas) [inline picard/script.py]: I wonder if we could make this even more generic, `anytag:*` could be used to delete all `anytag`.

Comment (samj1912) [inline picard/script.py]: We can make it more generic, it is also possible to introduce regex.

Comment (samj1912) [inline picard/script.py]: I chose the above ones just to include the common 'tag:name' type tags that we have.

Comment (zas) [inline picard/script.py]: I think it is fine as is for now. We'll see later if we want to extend this.

File: test/test_script.py [hunk 2]
```diff
@@ -324,4 +324,11 @@ def test_cmd_with_not_arguments(self):
             self.parser.eval("$noargstest()")
         except ScriptError:
             self.fail("Function noargs raised ScriptError unexpectedly.")
-        
+
+    def test_cmd_unset(self):
```
Comment (zas) [inline test/test_script.py]: There was no test for `unset()` ? :(
Can you please test the simple cases too (including the _/~ case) ?

Comment (samj1912) [inline test/test_script.py]: okay :)

Comment (samj1912) [inline test/test_script.py]: Done :)

**Pull request merged**